src/payload.rs

```diff
@@ -188,6 +220,17 @@ impl TryFrom<SubscribeTransaction> for TransactionPayload {
     }
 }
 
+impl TryFrom<Result<SubscribeTransaction, PayloadError>> for TransactionPayload {
+    type Error = PayloadError;
+
+    fn try_from(result: Result<SubscribeTransaction, PayloadError>) -> Result<Self, Self::Error> {
+        match result {
+            Ok(tx) => tx.try_into(),
+            Err(err) => Err(err),
+        }
+    }
+}
```

Comment:
I don't usually see results implement TryFrom. I would expect to see tryfrom for SubscribeTransaction into TransactionPayload.